hostinfo/packagetype_container.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//gobuild linux && ts_package_container
+
+package hostinfo
+
+func init() {
+	packageType = packageTypeContainer
```

Comment:
oh, actually we probably need to split this up from https://github.com/tailscale/tailscale/blob/60ccd190efc2cc010ac3ad2ce81e0518c7f9b440/hostinfo/hostinfo_linux.go#L21, so `packagetype_linux_nocontainer.go` and `packagetype_linux_container.go`

Reply:
whoops